features/taxation/applying_taxes_based_on_order_item_units_calculation_strategy/applying_correct_taxes_for_item_units_with_tax_rate_included_in_price.feature

```diff
@@ -0,0 +1,51 @@
+@applying_taxes
+Feature: Applying correct taxes for item units with tax rate included in price
+    In order to pay proper amount when buying goods with tax rate included in price
+    As a Visitor
+    I want to have correct taxes applied to my order
```

Comment:
As above 👆🏼

Reply:
I'm resolving these comments, as they litter the PRek a bit, the main comment is enough